huxley/www/js/components/AdvisorAssignmentsView.js

```diff
@@ -112,7 +114,11 @@ var AdvisorAssignmentsView = React.createClass({
                 </tr>
               </thead>
               <tbody>
-                {this.renderAssignmentRows()}
+                {
+                  Object.keys(committees).length > 0 && Object.keys(countries).length > 0 ?
```

Comment:
This is a side issue I've been noticing that seemed to small for a separate PR. The old code was triggering some promise rejections that didn't seem to be affecting anything but this provides a guarantee.